packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -1014,6 +1014,19 @@ const Account = Backbone.Model.extend(
       return this._fxaClient.attachedClients(this.get('sessionToken'));
     },
 
+    /**
+     * Check to see if the account has any active subscriptions.
+     *
+     * @returns {Promise} resolves to a bool.
+     */
+    hasActiveSubscriptions() {
+      return this.settingsData().then(
+        settingsData =>
+          Array.isArray(settingsData.subscriptions) &&
+          settingsData.subscriptions.some(s => s.status === 'active')
```

Comment:
I don't think we want this filter on the list - just a `settingsData.subscriptions.length > 0` should do.

I *think* [the `status` will equal `past_due` or `incomplete`](https://stripe.com/docs/api/subscriptions/object#subscription_object-status) in the case where the user's card has expired or been declined on an automatic payment. That's when [we display a warning on the settings page](https://github.com/mozilla/fxa/issues/970) and the user will need to click a link in the warning or the button to manage subscriptions and update their billing method. 

Technically, their subscription will be *not* active at that point, but I think we'll still want to offer access to the management pages.

Reply:
> Technically, their subscription will be not active at that point, but I think we'll still want to offer access to the management pages.

This seems reasonable so that users will be able to re-activate their subscription or update payment methods.